js/index.js

```diff
@@ -16,4 +16,44 @@ for (i=0; i < skills.length; i++) {
     skill.innerText = skills[i];
     skillsList.appendChild(skill)
 
-}
\ No newline at end of file
+}
+
+// messages event listener
+const messageForm = document.querySelector('form');
```

Comment:
This is fine; but per the instructions, we were asked to " Using "DOM Selection", select the "leave\_message" form by `name` attribute and store it in a variable named `messageForm`". So the DOM method for finding elements by their `name` attribute is: `getElementsByName`. This will return a list of nodes that match (which in this case we'll expect to have a length of 1 and just contain the one form with the `leave_message` name). 

https://developer.mozilla.org/en-US/docs/Web/API/Document/getElementsByName